plugins/salesforce-as-datasource-as-eventsource/README.md

```diff
@@ -1 +1,146 @@
-### plugin
\ No newline at end of file
+# godspeed-plugin-salesforce-as-datasource-as-eventsource
+
+Welcome to the [Godspeed](https://www.godspeed.systems/) salesforce Plugin! 🚀
+
+Salesforce is a powerful cloud-based customer relationship management (CRM) platform that enables businesses to manage their sales, marketing, and customer service operations efficiently. 
+
+A brief description of how to use Salesforce plug-in in our godspeed framework as Data Source as Event Source. 
+
```

Comment:
Mention as a note that this Salesforce plugin is both datasource (via its API) and an eventsource (to listen on salesforce events). Learn more about Eventsource-as-datasource type plugins here(link to that section in main documentation). As explained in the link shared just before, in order to use salesforce as an eventsource, you will need to setup the datasource as well. This plugin uses the salesforce Nodejs SDK (link to the nodejs sdk). If you want to call salesforce APIs via REST, then you should not use this plugin. For that you can use the axios plugin (insert link).